attest/ake/Cargo.toml

```diff
@@ -1,26 +1,29 @@
 [package]
-name = "attest-ake"
-version = "0.1.0"
+name = "mc-attest-ake"
+version = "0.1.3-alpha-1"
 authors = ["MobileCoin"]
 edition = "2018"
 
 [features]
 default = ["std"]
 std = [
-    "attest/std",
+    "mc-attest-core/std",
     "digest/std",
     "serde/std",
 ]
 sgx-sim = [
-    "attest/sgx-sim",
+    "mc-attest-core/sgx-sim",
 ]
 
+[lib]
+name = "attest_ake"
+
 [dependencies]
-attest = { path = "../../attest/core", default-features = false }
-common = { path = "../../common", default-features = false }
-keys = { path = "../../crypto/keys", default-features = false }
-mcnoise = { path = "../../crypto/ake/mcnoise", default-features = false }
-mcserial = { path = "../../util/mcserial", default-features = false }
+mc-attest-core = { version = "0.1.3-alpha-1", path = "../../attest/core", default-features = false }
```

Comment:
Actually looking at this, I think we should leave these version requirements off, what they're buying us is basically "if I introduce a breaking change and correctly bump semver of some crate in my repo, the build will fail", which is a pretty tame advantage to having 2400 lines of changes every time we want to bump our version, so I'd say just strip them entire and just use path like we were.

Sorry for sending you down the rabbit hole on that one.

Reply:
Np - removed the versions. How do you feel about leaving the versions at the top of the Cargo.toml (currently at 0.1.4-alpha-1)?

Reply:
In the `[package]` section? Those are required...

Reply:
Yeah, my question is whether to leave the changes from this PR, that set them all to 0.14-alpha-1.

Reply:
I think path dependencies should not include versions, though the package should. Additionally, we'll now need to make everything 0.1.5-alpha-1, because we've already tagged 0.1.4.

Reply:
SGTM! Thanks, @jcape